app/scripts/filters/resources.js

```diff
@@ -1029,32 +1029,66 @@ angular.module('openshiftConsole')
         return 'Terminating';
       }
 
-      var reason = pod.status.reason || pod.status.phase;
+      var initializing = false;
+      var reason;
 
-      // Print detailed container reasons if available. Only the last will be
-      // displayed if multiple containers have this detail.
-      angular.forEach(pod.status.containerStatuses, function(containerStatus) {
-        var containerReason = _.get(containerStatus, 'state.waiting.reason') || _.get(containerStatus, 'state.terminated.reason'),
-            signal,
-            exitCode;
+      _.forEach(pod.status.initContainerStatuses, function(initContainerStatus) {
```

Comment:
nit: We try to consistently use `_.each` (alias to `_.forEach`) so we don't have a mix of one and the other. They're the same, but if you don't know that, it might be confusing to see both in the code.